docs-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# June 10, 2022
+
+The App Services docs release notes start here from week ending May 27, 2022. Prior to this, the App Services docs release notes were presented alongside the Realm docs release notes in the [Realm Docs repository](https://github.com/mongodb/docs-realm/blob/master/docs-release-notes.md).
+
+## Manage & Deploy Apps
+- Deploy/Deployment Models & Regions
+  - Add new Azure deployment regions
+  - Specify regions tha support only local deployment
```

Comment:
```suggestion
```
We already did this, I only clarified the language for the intro :)

Reply:
The actual new regions PR was within this 2 week window, too. Downside of falling behind on the release notes.